ImplementationGuide/markdown/UebergreifendeFestlegungen/UebergreifendeFestlegungen_Suchparameter.md

```diff
@@ -22,7 +23,7 @@ Für die im Rahmen dieses Leitfadens relevanten Typen gelten folgende allgemeine
 
 ### Date/DateTime
 
-Die Präfixe `lt`,`le`,`gt`,`ge`,`eq` MÜSSEN für jeden Suchparameter vom Typ 'date/dateTime' unterstützt werden.
+Die Präfixe `lt`,`le`,`gt`,`ge`,`eq` KÖNNEN für jeden Suchparameter vom Typ 'date/dateTime' unterstützt werden.
```

Comment:
Verlieren wir damit nicht wichtige Funktionalität, um datums-eingeschränkt suchen zu können? Z.B. zeige alle Dokumente, die in den letzten zwei Monaten erstellt wurden?
Zeige alle Prozeduren, die nicht älter sind als 2 Jahre? 
Leistet das Fehlen dieser Funktion nicht der ungewollten Massendatenabfrage Vorschub, weil man zwangsläufig immer über den gesamten Zeitraum suchen muss?
Oder anders gefragt: was nützt die Datums-Suche, wenn man nur nach exakten Daten fragen kann? Sucht man hier nicht zu 99% in Zeit-*Räumen* statt in Zeit-*punkten*?

Ich könnte als Kompromiss mit der Einschränkung auf gt/lt gut leben, der Rest kann gerne optional bleiben.